egs/hub4_english/s5/local/tuning/run_segmentation_wsj_a.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
+set -e
+set -o pipefail
+
+# This script demonstrates how to re-segment long audios into short segments.
+# The basic idea is to decode with an existing in-domain acoustic model, and a
+# bigram language model built from the reference, and then work out the
+# segmentation from a ctm like file.
+
+%WER 19.1 | 728 32834 | 83.1 12.2 4.7 2.2 19.1 85.0 | exp/tri5_2a_cleaned/decode_nosp_eval97.pem_rescore/score_13_0.0/eval97.pem.ctm.filt.sys
```

Comment:
comment these WER results.
It's not really clear from just the listing of these results how to interpret these-- please add some comments to explain what's going on.